specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-07-01-preview/examples/Vulnerabilities/GetDeviceVulnerability.json

```diff
@@ -0,0 +1,41 @@
+{
+  "parameters": {
+    "api-version": "2021-07-01-preview",
+    "subscriptionId": "20ff7fc3-e762-44dd-bd96-b71116dcdc23",
+    "iotDefenderLocation": "eastus",
+    "deviceGroupName": "default",
+    "deviceId": "0418f897-937b-4cb5-92dc-dd31d3da7911",
+    "vulnerabilityId": "CVE-2017-7797"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/20ff7fc3-e762-44dd-bd96b71116dcdc23/providers/Microsoft.IoTSecurity/location/eastus/deviceGroups/default/devices/0418f897-937b-4cb5-92dc-dd31d3da7911/vulnerabilities/CVE-2017-7797",
+        "name": "CVE-2017-7797",
+        "type": "Microsoft.IoTSecurity/location/deviceGroups/devices/vulnerabilities",
+        "properties": {
+          "id": "CVE-2017-7797",
+          "title": "CVE-2017-7797",
```

Comment:
Is there a situation where this title doesn't match the resource name?

Reply:
I set the resource name with the id of the CVE.